lib/mix/tasks/dogma.ex

```diff
@@ -41,7 +41,7 @@ defmodule Mix.Tasks.Dogma do
       Reporters.default_reporter
     )
 
-    {List.first(files), reporter, noerror}
+    {List.first(files), reporter, noerror, read_stdin?}
```

Comment:
This tuple is starting to get a big unwieldy I think. Perhaps something we could fix in another pull request.